contract-tests/conftest.py

```diff
@@ -0,0 +1,44 @@
+# Configuration file for running contract-tests
+import pytest
+import requests
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--server",
+        dest="server",
+        default="http://localhost:8000",
+        help="Server to run tests against"
+    )
+    parser.addoption(
+        "--no-verify",
+        action="store_false",
+        dest="verify",
+        default=None,
+        help="Server to run tests against"
+    )
+
+
+@pytest.fixture
+def conf(request):
+    return request.config
+
+
+class RequestVerifyWrapper(object):
+    """
+    `request.Session` doesn't have a way to permanently change the verify flag. This provides that.
```

Comment:
Yeah, it felt like a weird omission so I gave the source code a quick search to find it.